gravitee-apim-portal-webui-next/src/app/catalog/catalog.component.ts

```diff
@@ -77,6 +79,14 @@ export class CatalogComponent {
   query: string = '';
   filter = signal('');
   filterAsCategory = computed(() => this.categories().find(cat => cat.id === this.filter()));
+  selectedFilterIndex = computed(() => {
+    const foundCategory = this.filterAsCategory();
+    if (foundCategory) {
+      const index = this.categories().indexOf(foundCategory);
+      return index + 1;
+    }
+    return 0;
```

Comment:
If you prefer, you can shorten it like this:
`return foundCategory ? this.categories().indexOf(foundCategory) + 1 : 0;`

Reply:
Done